server/drivers/hd44780-i2c.c

```diff
@@ -85,6 +85,10 @@
 
 #include "i2c.h"
 
+#ifdef HAVE_DEV_IICBUS_IIC_H
+#include <dev/iicbus/iic.h>
+#endif
+
 // Generally, any function that accesses the LCD control lines needs to be
```

Comment:
AFAICS the code depending on this header file is stale, so lets remove the code instead of including the header.